docs/content/concepts/metadata-tags/asset-metadata.mdx

```diff
@@ -0,0 +1,293 @@
+---
+title: "Defining asset metadata | Dagster Docs"
+description: "Learn to attach metadata to your assets to provide useful info and context."
+---
+
+# Asset metadata
+
+Attaching metadata to assets can help make your pipelines easy for you and other team members to understand. Data about your data (assets) can be attached to both asset definitions and materializations.
+
+By the end of this guide, you'll understand how to attach metadata to assets and view that metadata in the Dagster UI.
+
+---
+
+## How it works
+
+Attaching metadata to an asset is done using the `metadata` argument and a dictionary of key/value pairs. Keys must be a string, but values can:
+
+- Be any of the <PyObject object="MetadataValue" /> classes provided by Dagster
+- Primitive Python types, which Dagster will convert to the appropriate <PyObject object="MetadataValue" />
+
+How metadata is attached to an asset depends on the type of metadata being attached. Refer to the following sections for more details.
+
+---
+
+## Defining metadata
+
+There are two main types of metadata in Dagster:
+
+- **Definition metadata** is information that's fixed or doesn't frequently change. For example, definition metadata could be the expected columns an asset should produce, a link to its definition in GitHub, etc.
+- **Runtime, or materialization metadata** is information that changes after a materialization occurs. This could be how many records were processed, how long an asset took to materialize, etc.
```

Comment:
```suggestion
- **Runtime, or materialization metadata** is information that changes after a materialization occurs. This could be how many records were processed, or how long an asset took to materialize.
```